plotting_refactor/DataTreeItems.py

```diff
@@ -0,0 +1,25 @@
+from PySide6.QtWidgets import QTreeWidgetItem
+
+class PlotPageItem(QTreeWidgetItem):
+    def __init__(self, parent, name, fitpage_index, data_id):
+        super().__init__(parent, name)
+        self.fitpage_index = fitpage_index
+        self.data_id = data_id
+        super().setData(0, 1, self)
+
+    def get_fitpage_index(self):
+        return self.fitpage_index
+
+    def get_data_id(self):
+        return self.data_id
+
+class DataItem(PlotPageItem):
```

Comment:
These do not need to inherit from one another. The `DataItem` instances should be held within a `PlotPageItem` instead.